app.js

```diff
@@ -90,3 +90,48 @@ document.querySelector('.books-wrapper').addEventListener('click', (e) => {
   Book.deleteBook(e.target);
   Book.removeBooks(e.target.id);
 });
```

Comment:
- [ ] As specified, kindly ensure to use the Luxon library to implement the date found below the navigation bar.